games-arcade/digger/digger-2002.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="An arcade-style game similar to dig dug"
+DESCRIPTION="Digger was originally created by Windmill software in 1983 and \
+released as a copy-protected, bootable 5.25 floppy disk for the IBM PC. 
+As it requires a genuine CGA card, it didn't work on modern PCs.
+In 1998, I created Digger Remastered, which runs on all PCs with CGA \
+or better and plays just like the original. 
+It also has many new features, including:
+
+- Exit button
+- Optional VGA graphics
+- Recording and playback
+- Real time speed control
+- Keyboard redefinition
+- Gauntlet mode
+- Two player simultaneous mode"
+HOMEPAGE="https://digger.org"
+COPYRIGHT="1983 Windmill Software
+	1998 Andrew Jenner
+	2002 Maxim Sobolev"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://digger.org/digger-20020314.tar.gz"
+CHECKSUM_SHA256="e45cb1c5b67fba2fa1b768d1ed75153a64f82e73544c3d98eaf2c37b1dbefdc7"
+SOURCE_DIR="digger-20020314"
+PATCHES="digger-2002.patchset"
+ADDITIONAL_FILES="digger.rdef"
```

Comment:
Recipe name should reflect $portVersion (as shown in the link of my previous comment), so use the full date in the recipe name, eg `digger-20020314.recipe` and change the following lines accordingly:
```diff
SOURCE_URI="http://digger.org/digger-20020314.tar.gz"
CHECKSUM_SHA256="e45cb1c5b67fba2fa1b768d1ed75153a64f82e73544c3d98eaf2c37b1dbefdc7"
-SOURCE_DIR="digger-20020314"
-PATCHES="digger-2002.patchset"
ADDITIONAL_FILES="digger.rdef"
+SOURCE_URI="http://digger.org/digger-$portVersion.tar.gz"
CHECKSUM_SHA256="e45cb1c5b67fba2fa1b768d1ed75153a64f82e73544c3d98eaf2c37b1dbefdc7"
+PATCHES="digger-$portVersion.patchset"
ADDITIONAL_FILES="digger.rdef"
```
Patchset should be `digger-20020314.patchset` also then.
SOURCE_DIR shouldn't be needed then.